modules/command/src/main/java/org/apache/fluo/command/FluoProgram.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.command;
+
+import java.util.Optional;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterException;
+
+import static org.apache.fluo.command.FluoCommandType.CONFIG;
+import static org.apache.fluo.command.FluoCommandType.EXEC;
+import static org.apache.fluo.command.FluoCommandType.GET_JARS;
+import static org.apache.fluo.command.FluoCommandType.INIT;
+import static org.apache.fluo.command.FluoCommandType.LIST;
+import static org.apache.fluo.command.FluoCommandType.ORACLE;
+import static org.apache.fluo.command.FluoCommandType.REMOVE;
+import static org.apache.fluo.command.FluoCommandType.SCAN;
+import static org.apache.fluo.command.FluoCommandType.STATUS;
+import static org.apache.fluo.command.FluoCommandType.WAIT;
+import static org.apache.fluo.command.FluoCommandType.WORKER;
+
+public class FluoProgram {
```

Comment:
I'm now leveraging the command functionality form JCommander, which we weren't previously leveraging. You can read about it here [http://jcommander.org/#_more_complex_syntaxes_commands](http://jcommander.org/#_more_complex_syntaxes_commands).

From the Command Pattern I tried to make this class the Client class.